sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/implementation/ServiceBusReactorAmqpConnection.java

```diff
@@ -101,10 +117,14 @@ public ServiceBusReactorAmqpConnection(String connectionId, ConnectionOptions co
                     logger.info("Creating management node. entityPath: [{}]. address: [{}]. linkName: [{}]",
                         entityPath, address, linkName);
 
+                    TokenManager cbsBasedTokenManager =  new AzureTokenManagerProvider(
```

Comment:
Every time a peek operation is called, it'll create another management channel node.  This is a bit wasteful if we keep getting the management node and have to recreate it.

Reply:
I have tested with 4 peek() calls from one client , since we are storing `ServiceBusManagementNode` in `ConcurrentHashMap`  and return cached managementNode after once created, 

https://github.com/Azure/azure-sdk-for-java/pull/8983/files#diff-6460b86560d643d02d2e58af8b6213cfL96